x-pack/plugins/cloud_security_posture/common/types/rules/v3.ts

```diff
@@ -132,9 +132,7 @@ export interface FindCspBenchmarkRuleResponse {
 
 export const cspBenchmarkRules = schema.arrayOf(
   schema.object({
-    benchmark_id: schema.string(),
-    benchmark_version: schema.string(),
-    rule_number: schema.string(),
```

Comment:
btw why this change, deletion of `benchmark_id`, `benchmark_version` and `rule_number`? if I'm not missing smth it makes the comment in `bulk_action.ts` not match the reality anymore. Plus strictly speaking deleting attributes calls for a new version of the interface

Reply:
answering my own question regarding the version: as our APIs are internal and we do the change without the need to keep the old behavior on the frontend (as far as I can see there are no calls for bulk action with these 4 params yet), I guess we don't need the new version here. But I'm still not sure if we need to remove these params or not

Reply:
@CohenIdo I think I just don't fully understand what we use the `benchmark_id`, `benchmark_version` and `rule_number` stored on the saved object for. The confusion came from the fact that I thought that the combination of `benchmark_id`, `benchmark_version`, `rule_number` and `rule_id` is the unique identifier for the rule and we might want to disable/enable the same rules for different benchmark versions. We shouldn't block the PR on that I think, we can take it offline